crates/sdk/src/masp.rs

```diff
@@ -735,15 +737,11 @@ impl<U: ShieldedUtils + MaybeSend + MaybeSync> ShieldedContext<U> {
                 .block
                 .data;
 
-            for (idx, tx_event) in txs_results {
+            for idx in txs_results {
```

Comment:
This looks good to me, unfortunately we modified this file in the last release to require tx events for other things but ibc (basically to use the changed keys instead of `Transfer`) so we'll probably need to revert some of your changes here when merging